dali/operators/audio/resample.cc

```diff
@@ -96,6 +96,17 @@ type. Example::
 )",
     nullptr, false);
 
+// Deprecated alias
+DALI_SCHEMA(experimental__AudioResample)
```

Comment:
Note to reviewers: Please look at the rendered documentation too, when available